src/main/java/duke/helper/Ui.java

```diff
@@ -0,0 +1,34 @@
+package duke.helper;
+
+import java.util.Scanner;
+
+public class Ui {
+    Scanner sc = new Scanner(System.in);
+
+    public String readLine() {
+        return sc.nextLine();
+    }
+
+    /**
+     * helper method to print proper indentation.
+     *
+     * @param toPrint the message to be printed.
+     */
+    public void print(String toPrint) {
```

Comment:
Should you make the name for `print` more meaningful for its function which you nicely have described in your javadocs? 🤔 Perhaps something like `printWithIndentation`?